settings.gradle

```diff
@@ -6,7 +6,14 @@ include ':integration-tests:code-generator'
 include ':integration-tests:code-generator-tests'
 include ':integration-tests:dagger-factories-only'
 include ':integration-tests:library'
-include ':integration-tests:mpp:android-module'
+
+// TODO: There seems to be a bug with Kotlin 1.7 and MPP projects. Disable the Android
+//  module for now until the issue is resolved
```

Comment:
Ultra nit: extra space

Reply:
No, the extra space will make sure that IJ renders the multiline TODO as a single TODO.